roboteam_ai/src/world/Robot.cpp

```diff
@@ -90,6 +90,20 @@ void Robot::setGenevaState(unsigned char state) {
     else std::cout << "setting invalid geneva state (" << state << ") for robot with id " << id << std::endl;
 }
 
+const Robot Robot::clone() const {
```

Comment:
Robot newBot=oldBot::clone() is identical to Robot newBot=oldBot so this is a redundant function.